rgw/v2/utils/utils.py

```diff
@@ -287,6 +287,14 @@ def start(self):
         cmd = self.srv.cmd("start")
         return exec_shell_cmd(cmd)
 
+    def status(self):
+        """
+        starts the service
```

Comment:
can you change the doc string, this does not start the service, just gives status